src/ripple/nodestore/backend/RocksDBFactory.cpp

```diff
@@ -118,10 +118,15 @@ class RocksDBBackend
         options.env = env;
 
         if (keyValues.exists ("cache_mb"))
-            table_options.block_cache = rocksdb::NewLRUCache (get<int>(keyValues, "cache_mb") * 1024L * 1024L);
+            table_options.block_cache = rocksdb::NewLRUCache (
+                get<int>(keyValues, "cache_mb") * 1024L * 1024L);
 
         if (auto const v = get<int>(keyValues, "filter_bits"))
-            table_options.filter_policy.reset (rocksdb::NewBloomFilterPolicy (v));
+        {
+            bool filter_blocks = ! keyValues.exists ("filter_full") ||
```

Comment:
FYI: our official code style mandates a space after the `!`. Consider adding a comment to this PR if you want to change the style guide: https://github.com/ripple/rippled/pull/2204. (BTW, I prefer no space as well).

Reply:
I prefer the no space too.

Reply:
I hate to argue about style, but I'm going to anyway. :-)  I like to use the space to approximate binding precedence.  "No space" means "binds tightly" and "space" means "binds loosely".  So in this example since the `!` binds directly to `keyValues.exists ()`, no space.